### ISSUE TITLE
NTR: beta-alanine high in urine (Lab: chemistry: amino compounds)

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#8489]
New term request beta-alanine high in urine (segal_030430155123): Hyper-beta-alaninuria  UMLS: 1168194

**Comment by pnrobinson** [comment]
@MickeySegal
Adding new term: 
- ID: HP:6000276
- label: Elevated urinary beta-alanine level
- parents: [Increased urinary non-proteinogenic amino acid level (HP:0033098)](https://hpo.jax.org/app/browse/term/HP:0033098)
- pmid: PMID:7936305
- def: The amount of beta alanine in the urine, when normalized for urine concentration, is above the upper limit of normal.
- comment: 
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/8489
- synonyms: Hyper-beta-alaninuria(NONE)|Beta-alanine high in urine(NONE)

**Comment by pnrobinson** [comment]
@MickeySegal

We noticed that this term is a duplicate. I have obsoleted the above term. Please use the existing term

[Beta-alaninuria HP:0020079](https://hpo.jax.org/browse/term/HP:0020079)
